example/src/screens/HomeScreen.tsx

```diff
@@ -0,0 +1,34 @@
+import type { Subscription } from '@chargebee/react-native-chargebee';
+import React, { useEffect, useState } from 'react';
+import CoursesScreen from './CoursesScreen';
+import PurchasesScreen from './PurchasesScreen';
+
+const HomeScreen = ({ route, navigation }) => {
+  const [subscriptions, setSubscriptions] = useState<Array<Subscription>>([]);
+  const [customerId, setCustomerId] = useState<string>('');
+  const [isSubscribed, setIsSubscribed] = useState(false);
```

Comment:
Guess we can derive this state while rendering and don't need a separate state variable. What do you think?

Reply:
I will refactor this along with the next set of changes